README.md

```diff
@@ -1,4 +1,4 @@
-# ember-m3 [![Build Status](https://secure.travis-ci.org/hjdivad/ember-m3.svg?branch=master)](http://travis-ci.org/hjdivad/ember-m3)
+# Ember-m3 [![Build Status](https://secure.travis-ci.org/hjdivad/ember-m3.svg?branch=master)](http://travis-ci.org/hjdivad/ember-m3)
```

Comment:
@rwjblue I could go either way.  I take it you're in favour of always lowercase (even in title / start of sentence)?